dev-lua/luacheck/luacheck-0.23.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit toolchain-funcs
+
+DESCRIPTION="A tool for linting and static analysis of Lua code"
+HOMEPAGE="https://github.com/mpeterv/luacheck"
+SRC_URI="https://github.com/mpeterv/luacheck/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~x86"
```

Comment:
since one of the dependencies (lua-argparse) is not keyworded on `arm` `ppc` and `x86`, you have to drop those keywords here and file keywording bug and requirest keyword added to lua-argparse.